chipper/command_line.py

```diff
@@ -20,7 +18,10 @@ def run_analysis(input_directory, user_note_thresh, user_syll_sim_thresh,
     out_path : str
 
     """
-    files = glob.glob(os.path.join(input_directory, '*.gzip'))
+
+    files = [i for i in os.listdir(os.path.join(input_directory))
```

Comment:
@JamesPino did you mean for this to be just the file names and not the entire paths?

Reply:
Yep. I used os.path.join(input_directory, file_name) when i create file_names below. For some reason the way the code was set up it would make it path/path/file, which didn't actually work.